tripal_chado/tests/src/Kernel/Plugin/TripalBackendPublish/ChadoPublishTest.php

```diff
@@ -334,6 +335,26 @@ public function testTripalPublishServiceSingleJob() {
       $this->assertEquals($expected, $this->countFieldTable($field_table),
         'The drupal field table does not contain the expected number of publications for the analysis');
     }
+    // Tests unpublishing.
+    // Try to unpublish orphaned, but since there are no orphaned entities
+    // this should do nothing.
+    $publish_options = ['bundle' => 'organism', 'datastore' => 'chado_storage', 'schema_name' => $this->testSchemaName, 'unpublish' => TRUE];
+    $unpublished_entities = $this->chado_publish->publish($publish_options);
+    $this->assertCount(0, $unpublished_entities, 'No orphans, should not have unpublished any records');
+
+    // Delete one chado record and confirm that we can now unpublish it as
```

Comment:
For this one we will want to confirm that the drupal entity and field records have been deleted. No need to check chado here I suspect but you could do a full table count just to be safe.